server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceType.java

```diff
@@ -30,7 +30,8 @@
     ANY,
     NUMERIC,
     BYTES,
-    GEOPOINT;
+    GEOPOINT,
```

Comment:
Ditto to what I said to @not-napoleon in a different PR, we should add some tests to ensure these enum ordinals don't change.  

Since both you and Mark are altering this enum we should probably add that test (in an unrelated PR) before either PR merges :)

Reply:
++